src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/custom.py

```diff
@@ -0,0 +1,48 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# --locations
+# -ipRangeFilter
+# -defaultConsistencyLevel
+# - MaxIntervalInSeconds
+# - MaxStalenessPrefix
+# - resource-group-location
+# -Kind
+
+from azure.cli.command_modules.documentdb.sdk.models import (
+    ConsistencyPolicy,
+    DatabaseAccountCreateUpdateParameters,
+    DatabaseAccountKind,
+    DatabaseAccountOfferType,
+    DefaultConsistencyLevel,
+    Location
+)
+
+def cli_documentdb_create(client,
+                          resource_group_name,
+                          name,
+                          resource_group_location,
+                          locations,
+                          default_consistency_level=None,
+                          max_staleness_prefix=100,
+                          max_interval_in_seconds=5,
+                          ip_range_filter=None):
+    # pylint:disable=line-too-long
+    """Create new DocumentDB Database Account
+    :param resource_group_name: Name of resource group
+    :param name: Name of DocumentDB Database Account
+    :param location: Resource group location
```

Comment:
If these are in the params file, you don't need them here. These values will be ignored.